keras/engine/data_adapter.py

```diff
@@ -119,11 +119,11 @@ def get_dataset(self):
 
     Note that the dataset returned does not repeat for epoch, so caller might
     need to create new iterator for the same dataset at the beginning of the
-    epoch. This behavior might change in future.
+    epoch. This behavior might change in the future.
 
     Returns:
-      An tf.dataset.Dataset. Caller might use the dataset in different
-      context, eg iter(dataset) in eager to get the value directly, or in graph
+      A tf.dataset.Dataset. Caller might use the dataset in different
```

Comment:
- It's `tf.data.Dataset` (typo)
- Use backticks

Reply:
Done